components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRMService.java

```diff
@@ -335,6 +335,8 @@ public Application updateApplication(ApplicationUpdateRequest updateRequest, Str
                     appDTO.setTokenBindingType(OAuth2Constants.TokenBinderType.CERTIFICATE_BASED_TOKEN_BINDER);
                     appDTO.setTokenBindingValidationEnabled(true);
                 }
+            } else {
+                appDTO.setTokenBindingType("");
```

Comment:
DCR might be updating an already created application through the management API or the console so the app could be having different binding type as well. So we should check if the app already has a binding type and if not set the empty string